ai_edge_torch/generative/examples/stable_diffusion/convert_to_tflite.py

```diff
@@ -85,7 +89,7 @@ def convert_stable_diffusion_to_tflite(
   ).convert().export('/tmp/stable_diffusion/diffusion.tflite')
 
   # Image decoder
-  ai_edge_torch.signature('decode', decoder, (input_latents,)).convert().export(
+  ai_edge_torch.signature('decode', decoder_model, (input_latents,)).convert().export(
       '/tmp/stable_diffusion/decoder.tflite'
```

Comment:
have you compared if the generated tflite model is same with the previous one?

Reply:
Yes, it is verified that the refactored one has the same converted tflite model.